Tasks/ContainerBuildV0/src/utils.ts

```diff
@@ -100,7 +100,10 @@ export async function getServiceDetails() {
     };
     var serviceResponse = kubectlTool.execSync(executionOption);
 
-    if (serviceResponse && serviceResponse.stdout) {
+    if (serviceResponse && serviceResponse.stderr) {
+        tl.setResult(tl.TaskResult.Failed, serviceResponse.stderr);
```

Comment:
throw exception with 'serviceResponse.stderr' main function will handle and set task failed.

Reply:
Added throw exception